lib/media/track/remotemediatrack.js

```diff
@@ -45,16 +63,24 @@ function mixinRemoteMediaTrack(AudioOrVideoTrack) {
     /**
      * Whether the {@link RemoteMediaTrack} is subscribed to
      * @property {boolean}
+     * @deprecated
      */
     get isSubscribed() {
+      this._log.deprecated('RemoteMediaTrack#isSubscribed has been deprecated and is '
+        + 'scheduled for removal in twilio-video.js@2.0.0. Use the '
+        + 'parent RemoteTrackPublication\'s .isSubscribed instead.');
       return this._isSubscribed;
     }
 
     /**
      * The SID assigned to the {@link RemoteMediaTrack}.
      * @property {Track.SID}
+     * @deprecated
      */
     get sid() {
+      this._log.deprecated('RemoteMediaTrack#sid has been deprecated and is '
+        + 'scheduled for removal in twilio-video.js@2.0.0. Use the '
+        + 'parent RemoteTrackPublication\'s .sid instead.');
```

Comment:
This should not actually be deprecated. Perhaps we made a mistake in the implementation plan, but you can see here in [the IDL](https://code.hq.twilio.com/client/video-sdk-api/blob/master/idl/remotetrack.md) that `sid` is still included.